concepts/50 React Components/40 Component Configuration Syntax/55 Markup Customization/3 Using a Rendering Function.md

```diff
@@ -1,42 +1,71 @@
-In the following code, rendering functions are used to specify the [List](https://js.devexpress.com/Demos/WidgetsGallery/Demo/List/ListEditingAndAPI)'s [itemTemplate](/api-reference/10%20UI%20Components/dxList/1%20Configuration/itemTemplate.md '/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate') and the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/PredefinedTypes)'s [template](/api-reference/10%20UI%20Components/dxButton/1%20Configuration/template.md '/Documentation/ApiReference/UI_Components/dxButton/Configuration/#template'):
+The following example demonstrates a [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/) component which cells display a combination of grid's data and a [TextBox](/Documentation/Guide/UI_Components/TextBox/Getting_Started_with_TextBox/) value. For this task, a rendering function wrapped in [useCallback](https://react.dev/reference/react/useCallback) was used. This way you can pass the TextBox value to the column's [cellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellRender) property.
```

Comment:
The word "which" should actually be "whose" in this context ("components whose cells display ...").

And then I'm trying to keep everything active (remove passive voice). 

```suggestion
The following example customizes a [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/) component to display a combination of grid's data and a [TextBox](/Documentation/Guide/UI_Components/TextBox/Getting_Started_with_TextBox/) value within cells. The code wraps a rendering function within a [useCallback](https://react.dev/reference/react/useCallback) hook. This way you can pass the TextBox value to the column's [cellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellRender) property.
```